parm/config/config.wave

```diff
@@ -64,19 +74,24 @@ export FHINCP_WAV=`expr $DTPNT_WAV / 3600`
 # Selected output parameters (gridded)
 export OUTPARS_WAV="WND HS FP DP PHS PTP PDIR"
 
+# Options for point output (switch on/off boundary point output)
```

Comment:
Where is this coming from?

Reply:
See example config.wave I was provided: /work/noaa/global/wkolczyn/save/global-workflow/expdir/example/config.wave

Reply:
Let's remove this as it's commented out, let's just not move it in.

Reply:
Roger that, will remove, thanks!